lib/stream/promises.js

```diff
@@ -0,0 +1,3 @@
+'use strict';
+
+module.exports = require('internal/streams/promises');
```

Comment:
I would just move `internal/streams/promises` here and avoid having this file.

Reply:
done